Mission-Vision-Statement.md

```diff
@@ -0,0 +1,10 @@
+# Mission and Vision
+The mission of this project is to help increase diversity and inclusion (D&I) in the workplace using the open-source approach. We hope to deliver a framework that can be used by all organizations that want to implement, improve or validate their D&I efforts. We will leverage the entire community's knowledge and experience to build and test the framework that we propose. The open-source approach we will provide everyone better visibility of D&I efforts within organizations using our framework
```

Comment:
The mission of this project is to help increase diversity and inclusion (D&I) in the workplace using the open source approach. We hope to deliver a framework that can be used by all organizations that want to implement, improve, or validate their D&I efforts. We will leverage the entire community's knowledge and experience to build and test the framework that we propose. The open source approach we provide will lead to better visibility of D&I efforts within organizations using our framework.